### DISCUSSION TITLE
About dynamically update  Chart's data

### DISCUSSION THREAD

**Discussion by dlyltm** [dotnetcore/BootstrapBlazor#2273] [Q&A] [answered]
How to update Chart's data  dynamically ? At first, the chart is not loaded with data, and then dynamically add data to it. But in any case, the data line can't be displayed.

**Comment by ArgoZhang** [comment, marked as answer]
@dlyltm 

```
<Chart @ref="LineChart" />
```

1. use `@ref` to get the instance of `Chart` component
2. use instance method `Update` or `Reload`

**Reply by dlyltm** [reply]
It seems using Reload ok.
